badge.html

```diff
@@ -2,9 +2,28 @@
 <link rel="import" href="color.html">
 <link rel="import" href="typography.html">
 
+<!-- MAGI ADD START
+<link rel="import" href="../vaadin-themable-mixin/register-styles.html">
```

Comment:
This seems to be the only place where this is inside the `MAGI ADD`. In the other files it's outside (above) it. Is there a reason why it's not inside this everywhere?

Reply:
I think I figured out why it's like this. I think this file is ok, but see my review below regarding the other files.

Reply:
Thanks, done.